scheduler/src/main/java/gov/nasa/jpl/aerie/scheduler/SimulationFacade.java

```diff
@@ -71,7 +71,7 @@
   //simulation results from the last simulation, as output directly by simulation driver
   private SimulationResults lastSimDriverResults;
 
-  private Map<String, ActivityInstanceId> activityNameToActivityId= new HashMap<>();;
+  private Map<Long, ActivityInstanceId> activityNameToActivityId= new HashMap<>();;
```

Comment:
It has been renamed `activityIdToActivityInstanceId`.

Reply:
Please do tell me I'm confused, if you think I am 😄. I think this name doesn't yet accurately communicate this map's relation. Looking at the code, it seems to me that it's planActInstanceIdToSimulationActInstanceId or something similar.

Reply:
There is a third type of activity instances, the scheduling activity instances that are the subject of this PR. This container maps from scheduling activity instances ids to simulated activity instances ids. Do you want me to box the Activity Instance long id into a `SchedulingActivityInstanceId` to make that more clear ?

Reply:
> Please do tell me I'm confused, if you think I am 😄. I think this name doesn't yet accurately communicate this map's relation. Looking at the code, it seems to me that it's planActInstanceIdToSimulationActInstanceId or something similar.

Yes I can do that.

Reply:
I see… so there are really 3 different ID sources -
1. The scheduler autoincrements a counter for every ActivityInstance object it constructs
1. SimulationFacade generates ActivityInstanceIds for every simulation run
1. The simulation engine may assign new IDs to child activities that were not part of the plan

So yeah, I think it would be helpful to wrap the scheduler-specific id in a record :+1:

Reply:
There is now a `SchedulingActivityInstanceId` record and the name of the map has been changed to `planActInstanceIdToSimulationActInstanceId`.